README.rst

```diff
@@ -8,15 +8,31 @@ python-crfsuite
 .. image:: https://ci.appveyor.com/api/projects/status/uaq4sw8tc0oojr4v?svg=true
     :target: https://ci.appveyor.com/project/kmike/python-crfsuite-mhi5h
 
+.. image:: https://img.shields.io/github/release/scrapinghub/python-crfsuite.svg?style=flat-square
+    :target: https://github.com/scrapinghub/python-crfsuite/releases
+    :alt: Current Release Version
```

Comment:
I thought it was a caching issue. But I have confirmed it depends on the `releases` API which only returns proper releases (Going to releases > Tags > Add release notes (can be empty) > Publish).

I will remove this badge.